### PR TITLE
Update and test travis

### PR DESCRIPTION
**Pull request by RSully**
I want to see if these these previously skipped tests pass.
Also cleaned up the dependencies slightly.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,7 +31,9 @@ before_install:
 install:
   # Customise the testing environment
   # ---------------------------------
+  - conda install --channel ioos pytest
   - conda install --channel ioos --file requirements.txt
-  - conda install --channel ioos nco pyncml pytest pynco
+  - conda install --channel ioos pyncml pynco
```
Comment (ocefpaf) [inline .travis.yml]: Can you take the opportunity and change the channels here from `ioos` to `conda-forge`?

File: .travis.yml [hunk 2]
```diff
@@ -5,10 +5,6 @@ python:
   - "3.5"
 
 sudo: false
-addons:
-  apt:
-    packages:
-      - nco
```
Comment (ocefpaf) [inline .travis.yml]: :+1:

File: .travis.yml (hunk 1 above)
Comment (RSully) [inline .travis.yml]: What would you think of adding

```
conda config --add channels conda-forge
conda config --add channels ioos
```

to the `before_install` section and dropping the `--channel` here?

Comment (ocefpaf) [inline .travis.yml]: I :+1:  to that but I am not the main maintainer here. What do you prefer @kwilcox?

Comment (kwilcox) [inline .travis.yml]: Sounds good. I think we can completely drop the `ioos` channel, so just add `conda-forge`

**Pull request merged**